### PR TITLE
Users/markwolff/fetch e2e 636

### PR DESCRIPTION
**Pull request by markwolff**
Add end-to-end test (c828e58) for new automatic fetch monitoring from PR #636. Monitor is left off by default, or if browser does not natively support fetch (f0cc5aa).

### CODE REVIEW

**Comment by jpiyali** [comment]
@markwolff - can you please add a description to the PR?

**Comment by markwolff** [comment]
@jpiyali PR description added

File: JavaScript/JavaScriptSDK/ajax/fetch.ts [hunk 1]
```diff
@@ -39,7 +39,8 @@ module Microsoft.ApplicationInsights {
             let result: boolean = true;
             if (extensions.IsNullOrUndefined((window as any).Request) ||
                 extensions.IsNullOrUndefined((window as any).Request.prototype) ||
-                extensions.IsNullOrUndefined(window.fetch)) {
+                extensions.IsNullOrUndefined(window.fetch) ||
+                window.fetch.toString().indexOf("new XMLHttpRequest") !== -1) {
```
Comment (jpiyali) [inline JavaScript/JavaScriptSDK/ajax/fetch.ts]: Please add a comment on what this is intended for

Comment (markwolff) [reply JavaScript/JavaScriptSDK/ajax/fetch.ts]: Added in dac1870

**Pull request merged**